tip-router-operator-cli/src/stake_meta_generator.rs

```diff
@@ -0,0 +1,931 @@
+use {
```

Comment:
would like to keep this file together with the other logic that was pulled from jito-solana/tip-distributor. which is mainly: meta-merkle-tree/generated-merkle-tree. can we create a new module, that contains this this file and that one? naming is getting kind of weird, I think we can just call that one generated_merkle_tree but open to other ideas

Reply:
agree naming is getting hard to grok, but i don't have a better suggestion